### ISSUE TITLE
Chat v1.0 training recipe

### ISSUE THREAD

**Issue by normster** [jzhang38/TinyLlama#137]
Hi,

In the model card for the chat v1.0 model, it mentions following the "Zephyr training recipe". Does this mean using the [alignment-handbook](https://github.com/huggingface/alignment-handbook/tree/main/recipes) codebase or another reproduction of the Zephyr recipe?

Thanks!

**Comment by jzhang38** [comment]
Yes, we use the alignment handbook without changing any hyperparam.
